README.md

```diff
@@ -898,6 +907,21 @@ time to IPO has been increasing, and was **7.7 years** in **2016** compared to *
 🚧 What are the restrictions and regulations on selling stock that affect employees at
```

Comment:
@jlevy let me know if you did want me to draft up this section in its own PR or elsewhere.

Reply:
Feel free to put a draft paragraph in Notion, if you have it; but let's focus on production PRs now.

Reply:
👍